examples/bridge-app/linux/Dockerfile

```diff
@@ -0,0 +1,23 @@
+#
+#   Copyright (c) 2020 Project CHIP Authors
```

Comment:
2021

Since this is a bridge test, should the dockerfile include both a bridge and a light? Or is that supposed to be some cluster setup?

Reply:
The bridge device would have various endpoints. Each endpoint could be a lightbulb, switch or any other device type.